### ISSUE TITLE
Update polkadot apps lib

### ISSUE THREAD

**Issue by vikiival** [paraspell/xcm-sdk#32]
Hello,

currently paraspell has really old dependencies and it creates a big problem for dApps onboarding this sdk

```
➜  nft-gallery git:(main) ✗ pnpm why @polkadot/util@10.4.2
Legend: production dependency, optional only, dev only

dependencies:
@paraspell/sdk 0.0.25
├─┬ @polkadot/api 9.14.2
│ ├─┬ @polkadot/api-augment 9.14.2
│ │ ├─┬ @polkadot/api-base 9.14.2
│ │ │ ├─┬ @polkadot/rpc-core 9.14.2
│ │ │ │ ├─┬ @polkadot/rpc-augment 9.14.2
│ │ │ │ │ ├─┬ @polkadot/types 9.14.2
│ │ │ │ │ │ ├─┬ @polkadot/keyring 10.4.2
│ │ │ │ │ │ │ ├── @polkadot/util 10.4.2
│ │ │ │ │ │ │ └─┬ @polkadot/util-crypto 10.4.2
│ │ │ │ │ │ │   ├─┬ @polkadot/networks 10.4.2
```

Please update this repo to at least `10.2.2` thanks!

**Comment by dudo50** [comment]
Thank you for raising this issue! This will be worked on and resolved today.
With kind regards,
Dudo

**Comment by dudo50** [comment]
As mentioned in #33 @vikiival I will keep this issue open until we fix Apps lib or until Apps fixes their lib.

The package is now released as v1.1.2.

**Comment by vikiival** [comment]
Thanks ^-^
